client/packages/system/src/Immunisation/api/hooks/useVaccineCourse.ts

```diff
@@ -18,6 +18,16 @@ import { DraftVaccineCourse, DraftVaccineCourseItem } from './types';
 
 // export interface DraftVaccineCourse extends VaccineCourseFragment {}
 
+export enum UpdateVaccineCourseError {
+  DatabaseError = 'Database Error',
+  RecordProgramCombinationAlreadyExists = 'Course name already exists on this program',
+}
+
+export enum InsertVaccineCourseError {
+  RecordAlreadyExist = 'Record already exists',
+  RecordProgramCombinationAlreadyExists = 'Course name already exists on this program',
+}
+
```

Comment:
these don't need to be exported.
and the `DraftVaccineCourseSchedule` can be deleted, it's unused.